app/api/v2/views/redflag_views.py

```diff
@@ -66,23 +66,18 @@ def get(self, id):
                 }), 200)
 
         return {"error": ["Incident with that id not found"]}, 404
```

Comment:
why do you have this message in a list?

Reply:
Thank you for the feedback, I have fixed this, I've removed lists from all responses.